examples/models/gltf/BoomBox/README.md

```diff
@@ -0,0 +1,9 @@
+# Boom Box
+## Screenshot
+
+![screenshot](screenshot/screenshot.jpg)
+
+
+## License Information
+
+Donated by Microsoft for glTF testing.
```

Comment:
Not sure this is a viable license. @mrdoob do these models need to be under an MIT-compatible license as well?